### PR TITLE
upgrade simnet tests to v8

### PR DESCRIPTION
**Pull request by radupopa2010**
Upgrade tests to simnet v8. This new version uses config for tests that live in this repo: https://github.com/paritytech/substrate/tree/master/simnet_tests.

At the moment tests run on a nightly schedule, on the master branch. 
We have 2 kind of tests: "quick" and "long".
A "quick" test is a smoke test where basic check-expect tests run by checking values from metrics exposed by the app.
A "long" test is the load testing where we send 50K transactions into the network and check if all completed successfully.

We will do followup PR where we will run qucik test for all commits.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -740,13 +742,42 @@ trigger-simnet:
...
-    TRGR_REF:                      ${CI_COMMIT_REF_NAME}
-    # Simnet project ID
-    DWNSTRM_ID:                    332
-  script:
-    # API trigger for a Simnet job
-    - .maintain/gitlab/trigger_pipeline.sh --simnet-version=${SIMNET_REF}
+  # variables:
+  # `build.env` brings here `${SUBSTRATE_IMAGE_NAME}` and `${SUBSTRATE_IMAGE_TAG}`
+  # (`$VERSION` here, # i.e. `2643-0.8.29-5f689e0a-6b24dc54`).
+  # ${SIMNET_REF} is a gitlab variable
```
Comment (TriplEight) [inline .gitlab-ci.yml:748]: you could also print out the variables here as you did in polkadot

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -740,13 +742,42 @@ trigger-simnet:
...
+#   # ${SIMNET_REF} is a gitlab variable
+#   script:
+#     - /home/nonroot/simnet/gurke/scripts/run-test-environment-manager.sh
+#         --github-remote-dir=https://github.com/paritytech/substrate/tree/master/simnet_tests
+#         --config=simnet_tests/configs/default_local_testnet.toml
+#         --image="${SUBSTRATE_IMAGE_NAME}:${SUBSTRATE_IMAGE_TAG}"
+#         --features=simnet_tests/tests/quick/
+#   retry: 2
+#   tags:
+#     - parity-simnet
```
Comment (TriplEight) [inline .gitlab-ci.yml:783]: let's have this job in a separate PR when everything will be ready for it.

**Review by TriplEight — CHANGES_REQUESTED** [review]

**Comment by TriplEight** [comment]
please write a PR description as well

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -729,9 +729,11 @@ deploy-prometheus-alerting-rules:
         - .gitlab-ci.yml
         - .maintain/monitoring/**/*
 
-trigger-simnet:
+# runs quick and log tests on nightly schedule
```
[the remark this answers is not in the document]
Comment (radupopa2010) [reply .gitlab-ci.yml:732]: yeah, sorry typeo

File: .gitlab-ci.yml (hunk 2 above)
Comment (radupopa2010) [reply .gitlab-ci.yml:783]: I was just thinking the same, to do separate PRs

File: .gitlab-ci.yml (hunk 1 above)
Comment (radupopa2010) [reply .gitlab-ci.yml:748]: will do

**Review by TriplEight — CHANGES_REQUESTED** [review]

**Review by TriplEight — APPROVED** [review]
good to go. Please merge as pipeline gets green and ping me to change the version variable in gitlab.
Also, please note to change that comment.

**Review by grbIzl — APPROVED** [review]

File: .gitlab-ci.yml [hunk 4]
```diff
@@ -121,6 +121,7 @@ default:
     - if: $CI_PIPELINE_SOURCE == "schedule"
     - if: $CI_COMMIT_REF_NAME == "master"
     - if: $CI_COMMIT_REF_NAME =~ /^v[0-9]+\.[0-9]+.*$/              # i.e. v1.0, v2.1rc1
+      variables:
```
Comment (TriplEight) [inline .gitlab-ci.yml:124]: what does it do?

**Pull request merged**